worksheet_C/README.md

```diff
@@ -0,0 +1,17 @@
+(a) Checks if any items on the list are repeated
+
+(b) It's taking a value from the list O(n) and checking it against every other value of the list O(n^2) 
+
+(c) It still checks every item on the list against every other item
```

Comment:
True, but needs more explanation (i.e. explain that the original algorithm checks each pair in both possible orders, whereas the new one checks each pair only once)